### ISSUE TITLE
Does Fastor support FFT with FFTW or FFT in MKL?

### ISSUE THREAD

**Issue by xichaoqiang** [romeric/Fastor#125]
Hi,Sir. I am intrested in your package.

Does Fastor support FFT with FFTW or FFT in MKL?

How have not see any example code.

Can you share me？

**Comment by romeric** [comment]
No, Fastor does not have FFT/FFTW support at the moment. We will see if we can provide support through MKL in the near future.
